src/components/CategoryList.jsx

```diff
@@ -11,6 +11,7 @@ const handleCategoryClick = category => {
 const CategoryList = ({ className, limit = Number.MAX_VALUE }) =>
   <div className={className}>
     {CategoryService.getCategories()
+      .filter(c => c.iconName)
```

Comment:
:heart: for the code style but it may be useful to add a comment here or use a named function as filter argument.